packages/std/src/math/uint128.rs

```diff
@@ -191,6 +192,14 @@ impl TryFrom<&str> for Uint128 {
     }
 }
 
+impl FromStr for Uint128 {
+    type Err = StdError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        Self::try_from(s)
+    }
+}
```

Comment:
Good stuff. Can you add this to the CHANGELOG?

Also I wonder if we should swap the call order (`try_from` implemented via the more specific `from_str`).